presentation.md

```diff
@@ -2,23 +2,48 @@
 % Clara Casas Castedo & Miguel Sánchez de León Peque
 % 2018-02-16
 
-Introducción
-============
+---
+
+<video src="./videos/bulebule_camera_onboard.mp4" controls muted>
+</video>
```

Comment:
I think you do not need the HTML `<video>` tag. Look at how I have included videos (it is just like images). :wink: